src/energyplus/ForwardTranslator.cpp

```diff
@@ -3463,6 +3468,7 @@ namespace energyplus {
       IddObjectType::OS_Coil_Heating_Water,
       IddObjectType::OS_Coil_Heating_WaterToAirHeatPump_EquationFit,
       IddObjectType::OS_Coil_WaterHeating_Desuperheater,
+      IddObjectType::OS_Coil_UserDefined,
```

Comment:
That should not be there in the list of objects to always translate. I know you're following the existing pattern, but an AirLoop/PlantLoop/ZoneHVAC should be responsible for triggering the translation of its coil